Cannot build cobbler-api - Cannot find module 'typescript-xmlrpc' or its corresponding type declarations.

**Describe the bug**
This is my first time trying to build this project and it is failing for me.  It appears that there is a circular dependency that isn't mentioned in the readme.

**To Reproduce**
Steps to reproduce the behavior:
1. `npm install`
2. `npm run build cobbler-api`

**Expected behavior**
Build succeeds

**Screenshots**
```
cobbler-web (main)]$ npm run build cobbler-api
> cobbler@0.0.0 build
> ng build cobbler-api

Building Angular Package

------------------------------------------------------------------------------
Building entry point 'cobbler-api'
------------------------------------------------------------------------------
✖ Compiling with Angular sources in Ivy full compilation mode.
projects/cobbler-api/src/lib/cobbler-api.service.ts:4:92 - error TS2307: Cannot find module 'typescript-xmlrpc' or its corresponding type declarations.

4 import {AngularXmlrpcService, MethodResponse, MethodFault, XmlRpcStruct, XmlRpcArray} from 'typescript-xmlrpc';
                                                                                             ~~~~~~~~~~~~~~~~~~~

cobbler-web (main)]$ npm run build typescript-xmlrpc
> cobbler@0.0.0 build
> ng build typescript-xmlrpc

Building Angular Package

------------------------------------------------------------------------------
Building entry point 'typescript-xmlrpc'
------------------------------------------------------------------------------
✔ Compiling with Angular sources in Ivy full compilation mode.
✔ Generating FESM2020
✔ Generating FESM2015
✔ Copying assets
✔ Writing package manifest
✔ Built typescript-xmlrpc

------------------------------------------------------------------------------
Built Angular Package
 - from: /home/orion/cobbler-web/projects/typescript-xmlrpc
 - to:   /home/orion/cobbler-web/dist/typescript-xmlrpc
------------------------------------------------------------------------------

Build at: 2023-10-24T02:22:26.509Z - Time: 5114ms
cobbler-web (main)]$ ./node_modules/.bin/ng serve
✔ Browser application bundle generation complete.

Initial Chunk Files   | Names         |  Raw Size
vendor.js             | vendor        |   4.52 MB | 
polyfills.js          | polyfills     | 300.00 kB | 
styles.css, styles.js | styles        | 251.98 kB | 
main.js               | main          | 203.95 kB | 
runtime.js            | runtime       |   6.53 kB | 

                      | Initial Total |   5.27 MB

Build at: 2023-10-24T02:24:04.789Z - Hash: ff587ad2cf9fdea4 - Time: 18148ms

./projects/cobbler-frontend/src/app/app.module.ts:20:0-61 - Error: Module not found: Error: Can't resolve 'cobbler-api' in '/home/orion/cobbler-web/projects/cobbler-frontend/src/app'

./projects/cobbler-frontend/src/app/login/login.component.ts:5:0-42 - Error: Module not found: Error: Can't resolve 'cobbler-api' in '/home/orion/cobbler-web/projects/cobbler-frontend/src/app/login'

Error: projects/cobbler-frontend/src/app/app.module.ts:19:46 - error TS2307: Cannot find module 'cobbler-api' or its corresponding type declarations.

19 import {COBBLER_URL, cobblerUrlFactory} from 'cobbler-api';
                                                ~~~~~~~~~~~~~


Error: projects/cobbler-frontend/src/app/app.module.ts:58:5 - error NG6001: The class 'LogInFormComponent' is listed in the declarations of the NgModule 'AppModule', but is not a directive, a component, or a pipe. Either remove it from the NgModule's declarations, or add an appropriate Angular decorator.

58     LogInFormComponent,
       ~~~~~~~~~~~~~~~~~~

  projects/cobbler-frontend/src/app/login/login.component.ts:14:14
    14 export class LogInFormComponent {
                    ~~~~~~~~~~~~~~~~~~
    'LogInFormComponent' is declared here.


Error: projects/cobbler-frontend/src/app/app.module.ts:85:5 - error NG6001: The class 'NavbarComponent' is listed in the declarations of the NgModule 'AppModule', but is not a directive, a component, or a pipe. Either remove it from the NgModule's declarations, or add an appropriate Angular decorator.

85     NavbarComponent,
       ~~~~~~~~~~~~~~~

  projects/cobbler-frontend/src/app/navbar/navbar.component.ts:17:14
    17 export class NavbarComponent {
                    ~~~~~~~~~~~~~~~
    'NavbarComponent' is declared here.


Error: projects/cobbler-frontend/src/app/login/login.component.ts:4:46 - error TS2307: Cannot find module 'cobbler-api' or its corresponding type declarations.

4 import {CobblerApiService, COBBLER_URL} from 'cobbler-api';
                                               ~~~~~~~~~~~~~


Error: projects/cobbler-frontend/src/app/login/login.component.ts:43:13 - error NG2003: No suitable injection token for parameter 'cobblerApiService' of class 'LogInFormComponent'.
  Consider using the @Inject decorator to specify an injection token.

43     private cobblerApiService: CobblerApiService,
               ~~~~~~~~~~~~~~~~~

  projects/cobbler-frontend/src/app/login/login.component.ts:43:32
    43     private cobblerApiService: CobblerApiService,
                                      ~~~~~~~~~~~~~~~~~
    This type does not have a value, so it cannot be used as injection token.


Error: projects/cobbler-frontend/src/app/navbar/navbar.component.ts:6:33 - error TS2307: Cannot find module 'cobbler-api' or its corresponding type declarations.

6 import {CobblerApiService} from 'cobbler-api';
                                  ~~~~~~~~~~~~~


Error: projects/cobbler-frontend/src/app/navbar/navbar.component.ts:30:13 - error NG2003: No suitable injection token for parameter 'cobblerApiService' of class 'NavbarComponent'.
  Consider using the @Inject decorator to specify an injection token.

30     private cobblerApiService: CobblerApiService
               ~~~~~~~~~~~~~~~~~

  projects/cobbler-frontend/src/app/navbar/navbar.component.ts:30:32
    30     private cobblerApiService: CobblerApiService
                                      ~~~~~~~~~~~~~~~~~
    This type does not have a value, so it cannot be used as injection token.


Error: projects/cobbler-frontend/src/app/services/item-settings.service.ts:2:33 - error TS2307: Cannot find module 'cobbler-api' or its corresponding type declarations.

2 import {CobblerApiService} from 'cobbler-api';
                                  ~~~~~~~~~~~~~


Error: projects/cobbler-frontend/src/app/services/item-settings.service.ts:5:24 - error TS2307: Cannot find module 'cobbler-api' or its corresponding type declarations.

5 import {Settings} from 'cobbler-api';
                         ~~~~~~~~~~~~~




** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **


✖ Failed to compile.
```

**Desktop (please complete the following information):**
 - OS: Fedora
 - Version Rawhide

**Additional context**
If I run `npm run build cobbler-api` the build succeeds.  Is the order of the commands reversed in the readme?

--- a/README.md
+++ b/README.md
@@ -15,8 +15,8 @@ Initial developer: [@nelliesnoodles](https://github.com/nelliesnoodles) & [@Scho
 * build required projects
 
 ```shell
-npm run build cobbler-api
 npm run build typescript-xmlrpc
+npm run build cobbler-api
 ```
 
 ## Development server

